cmd/relay/main.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+
+	dhttp "github.com/drand/drand/http"
+	drand "github.com/drand/drand/protobuf/drand"
+
+	"github.com/urfave/cli/v2"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+var listenFlag = &cli.StringFlag{
+	Name:  "bind",
+	Usage: "local host:port to bind the listener",
+}
+
+var connectFlag = &cli.StringFlag{
+	Name:  "connect",
+	Usage: "host:port to dial to a GRPC drand public API",
+}
+
+var certFlag = &cli.StringFlag{
+	Name:  "cert",
+	Usage: "file containing GRPC transport credentials of peer",
+}
+
+// Relay a GRPC connection to an HTTP server.
+func Relay(c *cli.Context) error {
+	if !c.IsSet(connectFlag.Name) {
+		return fmt.Errorf("A 'connect' host must be provided")
+	}
+
+	opts := []grpc.DialOption{}
+	if c.IsSet(certFlag.Name) {
+		creds, _ := credentials.NewClientTLSFromFile(c.String(certFlag.Name), "")
+		opts = append(opts, grpc.WithTransportCredentials(creds))
+	} else {
+		opts = append(opts, grpc.WithInsecure())
```

Comment:
It might just be that the drand nodes is behind a regular TLS cert and that we dont have its cert on our hand now so it could look like 
```go		
config := &tls.Config{}
opts = append(opts, grpc.WithTransportCredentials(credentials.NewTLS(config))
```

Reply:
when you provide an empty config like this you end up with a self-signed certificate?

Reply:
No - using an empty TLS config like this makes gRPC uses the local OS cert store